css/style.css

```diff
@@ -0,0 +1,722 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:wght@900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=PT+Sans+Narrow:wght@700&family=Playfair+Display:wght@900&display=swap');
+:root {
+  --primary: #0699A6;
+  --secondary:  rgba(158, 140, 108, 0.2);
+  --black:black;
+  --white: #fff;
+  --gold: #9E8C6C;
+  --font-narrow: 'PT Sans Narrow';
+  --font-playfair-display: 'Playfair Display';
+  --font-serif: 'serif';
+  --font-sans-serif: 'sans-serif'
+}
+
+header{
+  height: 660px;
+  background: var(--secondary);
```

Comment:
`var(--secondary)` is okay, but maybe its not so clear that its a color. Maybe you want it to be `--green-secondary` or `--thicker-green` or something more explicit. That way your code reads nicely and cleanly.